systemtest/src/test/resources/upgrade/BundleDowngrade.yaml

```diff
@@ -46,7 +46,7 @@
     status: stable
     flakyEnvVariable: none
     reason: Test is working on all environment used by QE.
-  featureGatesBefore: "-KafkaNodePools,+UnidirectionalTopicOperator"
```

Comment:
Could you please update also `BundleUpgrade.yaml`? I'm thinking if we should upgrade from `+UnidirectionalTopicOperator` to `-UnidirectionalTopicOperator` or vice versa. Thanks!

Reply:
Thanks. There is no need to enable it on main, so I'll just remove +UTO FG. Let me know if this is fine.